osu.Game/Skinning/ISkinSerialisable.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+
+namespace osu.Game.Skinning
+{
+    /// <summary>
+    /// Denotes a drawable component which should be serialised as part of a skin.
+    /// Use <see cref="ISkinnableComponent"/> for components which should be mutable by the user / editor.
+    /// </summary>
+    public interface ISkinSerialisable : IDrawable
```

Comment:
I don't really understand this distinction between `ISkinSerialisable` and `ISkinnableComponent`, and the xmldoc doesn't explain it. Why does `ISkinSerialisable` need to exist, and `ISkinnableComponent` can't be used? They're both empty interfaces.
More pragmatically, a drawable which "can be adjusted via skinning specifications" (`ISkinnableComponent`) must be serialised anyway, no?

Reply:
`ISkinSerialisable` is serialised but ignored by the editor.

Reply:
Can't you add an `IsEditable` property? Using interfaces as a tagging mechanism like this always leads to confusion.

Reply:
Yep, almost went in that direction, but it mean that this will need to be added to each skin component. If you prefer that then I'll make the change.

For reference:

![20210513 164543 (JetBrains Rider)](https://user-images.githubusercontent.com/191335/118095128-abcddc80-b40a-11eb-9b21-35e519285144.png)

Reply:
You can implement a default member for the interface:
```csharp
public interface ISkinnableComponent
{
    bool IsEditable => true;
}
```
Because overrides to false are only done in two particular places, ever.